language/move-lang/src/shared/mod.rs

```diff
@@ -210,6 +212,51 @@ pub fn verify_and_create_named_address_mapping(
 
     Ok(mapping)
 }
+//#[derive(Ord, PartialOrd, Eq, PartialEq, Hash, Clone, Copy)]
```

Comment:
Fun that these can't be derived due to the format :'( Might want to remove this comment.

Reply:
ah, missed this. Thanks!